tests/src/Kernel/fixtures/markup_rendering_patterns/date_block.yml

```diff
@@ -6,10 +6,13 @@ date_block_default:
       date: '2021-05-26'
   assertions:
     count:
-      'time.bcl-date-block.d-flex.flex-column.align-items-center.justify-content-center.bg-date.text-light.pt-2.rounded-top.mw-date': 1
-      'time[datetime="2021-05-26"]': 1
-      'span.py-4.bg-white.border.border-bottom-0.w-100.fs-5.fw-medium.text-dark.text-center.text-uppercase': 1
-      'span.bg-lighter.border.w-100.text-center.fs-5.text-dark.rounded-bottom.mb-n1': 1
+      'time': 1
+      'time.bcl-date-block[datetime="2021-05-26"]': 1
+      'time > *': 1
+      'time > * > *': 2
+      'time > * > * > *': 1
+      'time > * > * > * > *': 0
+      'time > * > *:nth-child(2) > *': 0
```

Comment:
These counts verify there are no unexpected elements.
TBD.

Reply:
honestly I think we where checking to many classes previously but now it looks to cryptic, can u amend this to I recognize the date structure, or at least leave a comment as to what I'm seeing here